lib/public/css/base.scss

```diff
@@ -185,3 +185,11 @@ Styles for the 404 page
     color: white;
     padding-top: 20vh;
 }
+
```

Comment:
it will be nice if we also change using SCSS syntax

```suggestion
.lead {
 a {
  color: #007bff;
  &:hover {
  color: #ffffff;
  }
 }
}
```

Reply:
Sure. Will be in a few moments 🙂